packages/chess_com_api/chess_com_api.1.0.1/opam

```diff
@@ -0,0 +1,43 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis: "Chess.com API client for chess games"
+description:
+  "A client library for the Chess.com API that provides easy access to chess games, player data, tournament information, and daily puzzles. Supports fetching games in PGN format and extracting metadata."
+maintainer: ["ckaf borisnovoselov602@gmail.com"]
+authors: ["ckaf borisnovoselov602@gmail.com"]
+license: "MIT"
+tags: ["chess" "chess.com" "api" "http" "client" "games" "pgn"]
+homepage: "https://github.com/ckaf/pgn_parser"
+doc: "https://github.com/ckaf/pgn_parser"
+bug-reports: "https://github.com/ckaf/pgn_parser/issues"
+depends: [
+  "ocaml" {>= "4.14"}
+  "dune" {>= "3.19" & >= "3.0"}
+  "cohttp-lwt-unix" {>= "5.0"}
+  "yojson" {>= "1.7"}
+  "lwt" {>= "5.6"}
+  "lwt_ppx" {>= "2.0"}
+  "qcheck" {>= "0.21"}
+  "odoc" {with-doc}
+]
+build: [
+  ["dune" "subst"] {dev}
+  [
+    "dune"
+    "build"
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/ckaf/pgn_parser.git"
+x-maintenance-intent: ["(latest)"]
+
```

Comment:
```suggestion
available: arch != "arm32" & arch != "x86_32"
```
like for the 1.0.0 release as this also fails on 32-bit with `failed: Integer literal exceeds the range of representable integers of type int`